CHANGES.rst

```diff
@@ -1179,6 +1181,9 @@ New Features
 
 - ``astropy.io.fits``
 
+  - Fix convenience functions (`getdata`, `getheader`, `append`, `update`) to
```

Comment:
Good catch, thanks ! (and fixed) 
Not sure either how it works for bug fixes that affect the 2 previous versions.